WcaOnRails/spec/controllers/competitions_controller_spec.rb

```diff
@@ -213,6 +213,12 @@
         sign_in organizer
       end
 
+      it 'cannot confirm competition' do
+        patch :update, id: competition, competition: { name: competition.name }, commit: "Confirm"
```

Comment:
A different route for confirming competitions would slightly clean up the controller and make this spec more "safe" (a typo in the `commit` value would probably lead to a passing test).